alf/algorithms/lagrangian_reward_weight_algorithm.py

```diff
@@ -59,6 +69,9 @@ def __init__(self,
                 its init value is always used.
             optimizer (optimizer): optimizer for learning the reward weights.
             init_weights (float|list[float]): the initial reward weights.
+            max_weight (float): the reward weights will be clipped up to this value
+            reward_weight_projection (bool): whether project the weights to
```

Comment:
reward_weight_normalization might be a more descriptive name than reward_weight_projection